runtime/doc/vim_diff.txt

```diff
@@ -38,7 +39,13 @@ these differences.
 - 'wildmode' defaults to "list:longest,full"
 
 ==============================================================================
-3. Changed features					 *nvim-features-changed*
+3. New keymaps							  *nvim-keymaps*
+
+A small set of mappings have been changed in nvim's default configuration.
+However, these can be disabled by the user (see |nvim-extra-keymaps|).
```

Comment:
There are only four mappings, so perhaps they could be summarized here? Maybe something like bullet points, as is done below.